Fix events - Move attempt* events from question to assessment namespace, add in missing id field for question:scoreSet (#26)
ucfopen/Obojobo-Docs: releases/v3.3.2/developers/events/obojobo_events.md

Obojobo Events are used to announce and record actions.
They may be initiated on the server or client, and can be listened to in order to trigger other actions.

> In most cases, these events are useful for research data analysis. In which case, researchers are likely to receive a csv full of events. This document outlines what those events mean.

Reviewer: Maybe reword for clarity?
```suggestion
These events can be useful for research and data analysis. This document provides details about the different events.
```

| visitId | id of the visit created
| deactivatedVisitId | id of the deactivated visit

### *visit:start*

> Occurs after a visit is created, marking the visit as actually starting. Not all visits that are created are started.

Reviewer: Maybe reword for clarity?
```suggestion
Occurs when a visit is started after its creation. Not all visits that are created are started.
```
